src/sentry/auth/providers/google/templates/sentry_auth_google/configure.html

```diff
@@ -0,0 +1,3 @@
+<h3>Domain</h3>
```

Comment:
Would be nice to see this template live with other templates, but this is fine 🤷‍♂️

Reply:
hrm actually how does this possibly work....

Reply:
imo something we can follow up on - theres improvements we need to make to SSO for onpremise at some point anyways